index.html

```diff
@@ -0,0 +1,208 @@
+              <div class="services-items-div-leftside">
+                <div class="reviewer-image">
+                  <img src="assets/icons/reviewer.png" alt="" />
+                </div>
+                <span class="reviewer-position">Reviews</span>
+              </div>
+              <div class="services-items-div-rigthside">
+                <p class="reviewer-description">
+                  Get support from top talented code reviewers 24/7
+                </p>
+              </div>
+            </div>
+            <div class="services-items-div">
+              <div class="services-items-div-leftside">
+                <div class="reviewer-image">
+                  <img src="assets/icons/developer.png" alt="" />
+                </div>
+                <span class="reviewer-position">Developers</span>
+              </div>
+              <div class="services-items-div-rigthside">
+                <p class="reviewer-description">
+                  Hire or order pair programming session
+                </p>
+              </div>
+            </div>
+            <div class="services-items-div">
+              <div class="services-items-div-leftside">
+                <div class="reviewer-image">
+                  <img src="assets/icons/offer.png" alt="" />
+                </div>
+                <span class="reviewer-position">Offers</span>
+              </div>
+              <div class="services-items-div-rigthside">
+                <p class="reviewer-description">
+                  Apply to thousands of available offers
+                </p>
+              </div>
+            </div>
+            <div class="services-items-div">
+              <div class="services-items-div-leftside">
+                <div class="reviewer-image">
+                  <img src="assets/icons/project.png" alt="" />
+                </div>
+                <span class="reviewer-position">Projects</span>
+              </div>
+              <div class="services-items-div-rigthside">
+                <p class="reviewer-description">
+                  Contribute to open source projects
+                </p>
+              </div>
+            </div>
+            <div class="services-items-div">
+              <div class="services-items-div-leftside">
+                <div class="reviewer-image">
+                  <img src="assets/icons/forum.png" alt="" />
+                </div>
+                <span class="reviewer-position">Forum</span>
+              </div>
+              <div class="services-items-div-rigthside">
+                <p class="reviewer-description">
+                  Learn more about ReviewMe community
+                </p>
+              </div>
+            </div>
+          </div>
+
+          <div class="buttont-join-reviewme">
+            <a href="#">JOIN ReviewMe</a>
+          </div>
+          <div class="button-all-service">
+            <a href="#">SEE ALL SERVICES</a>
+          </div>
+        </div>
+      </section>
```

Comment:
- [ ] Kindly make sure to follow the design guidelines as much as possible, for this section, kindly address the following:-

    - You need to use the proper font, and font size to match the actual design.
    -  You need to add the proper paddings, spacings, and margins to adjust the layout.
    - The `Service` title should be adjusted and laid out in the middle of the section.
    - You should add the hover effect on each card as shown in the actual design.

| Your design | Actual design |
| ------------- | ------------- |
| ![image](https://github.com/kambereBr/ReviewMe/assets/102612396/d934f9a4-aabc-4586-89a8-b39e6d9d5433)  | ![image](https://github.com/kambereBr/ReviewMe/assets/102612396/2a98c305-1888-49b5-83a8-bdf39dc83241) |